docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -50,18 +48,22 @@ For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_H
 
 {% include Configuring-Alluxio-with-HDFS/bootstrapConf.md %}
 
-Alternatively, you can also create the configuration file from the template and set the contents manually. 
+Alternatively, you can also create the configuration file from the template and set the contents manually.
 
 {% include Common-Commands/copy-alluxio-env.md %}
 
-Then edit `alluxio-site.properties` file to set the under storage address to the HDFS namenode address
-(e.g., `hdfs://localhost:9000` if you are running the HDFS namenode locally with default port).
+Then edit `conf/alluxio-site.properties` file to set the under storage address to the HDFS namenode address
+and the HDFS directory you want to mount to Alluxio. For example, the under storage address can be
+`hdfs://localhost:9000` if you are running the HDFS namenode locally with default port and mapping HDFS root directory to Alluxio,
+or `hdfs://localhost:9000/alluxio/data` if only the HDFS directory `/alluxio/data` is mapped to Alluxio.
 
 {% include Configuring-Alluxio-with-HDFS/underfs-address.md %}
 
-Copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
+## HDFS configuration files 
+Please copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
```

Comment:
"conf" -> "configuration" ?

Also, could you add a little bit as to why we should copy those files?